src/lib/components/Ideogram.react.js

```diff
@@ -13,6 +13,9 @@ import {omit} from 'ramda';
  * by side tracks to show important genomic data.
  *
  * Go here to see it in action: https://eweitz.github.io/ideogram/
+ * Reference of the component's props :
+ * https://github.com/eweitz/ideogram/blob/
+ * 770ce1fe5ae4f27dd435f9f99948118af88f8349/api.md
```

Comment:
Oh, so it wasn't line-wrapping, the URL is over two lines in the source. Why don't we keep it over one line?

Reply:
@mkcor Because it is more than 100 characters then

Reply:
Isn't it 91 though?

Reply:
Yes it is, it was because my PyCharm switched to 80 line wrap :)